### ISSUE TITLE
Useful Variables not available in listener mode

### ISSUE THREAD

**Issue by damies13** [damies13/rfswarm#84]
**Describe the bug**

In the documentation for [Useful Variables](https://github.com/damies13/rfswarm/blob/master/Doc/Preparing_for_perf.md#Useful-Variables) states that variables for index, vuser and iteration are available, however this is only the case for xml mode and not in listener mode

**To Reproduce**
1. Add reference to one of these variable in you test
2. run test with rfswarm in listener mode

**Expected behavior**
variable should be populated with value from rfswarm

**Desktop (please complete the following information):**
 - OS: ALL
 - Version ALL

**Additional context**

rfswarm_agent.py:
```
	def runthread(self, jobid):

		...

		cmd.append("-M agent:{}".format(self.agentname))
		if self.xmlmode:
			cmd.append("-v index:{}".format(self.jobs[jobid]["ScriptIndex"]))
			cmd.append("-v vuser:{}".format(self.jobs[jobid]["VUser"]))
			cmd.append("-v iteration:{}".format(self.jobs[jobid]["Iteration"]))
		else:
			cmd.append("-M debuglevel:{}".format(self.debuglvl))
			cmd.append("-M index:{}".format(self.jobs[jobid]["ScriptIndex"]))
			cmd.append("-M vuser:{}".format(self.jobs[jobid]["VUser"]))

```

**Comment by damies13** [comment]
I noticed this bug when reviewing the code, probably low urgency but quite easy to fix when i'm back in the main branch.

**Comment by damies13** [comment]
Used these 2 lines in test case to verify working
```
	${variables} =	Get Variables
	Log    ${variables}
```

**Comment by damies13** [comment]
This has been resolved and merged into v0.6.6 branch

**Comment by damies13** [comment]
release v0.6.6 moved to v0.7.0
